### ISSUE TITLE
Page - update examples to provide aria-label/labelledby to page sections

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#6431]
**Describe the enhancement or change**
Examples and full screen demos where the page section component is being used (with the `<section>` element) should be updated to provide an aria-labelledby (if a visible text heading exists in the section) or aria-label (if no visible text heading exists). Context can be found in the following comment chain: https://github.com/patternfly/patternfly-org/pull/3880#discussion_r1516298378

Completing thia and the React counterpart will go towards adding in verbiage from the following [commit in the Page a11y docs](https://github.com/patternfly/patternfly-org/pull/3880/commits/3005c20de6d80b19e1adaa647d85f21ef7eab15f)

**Is this request originating from a Red Hat product team? If so, which ones and is there any sort of deadline for this enhancement?**

**Any other information?**
